src/main/kotlin/data/Project.kt

```diff
@@ -0,0 +1,9 @@
+package data
+
+data class Project(
+        var id: Int,
+        val targetLanguage: Language,
+        val sourceLanguage: Language,
+        var title: String,
+        val collection: Collection
```

Comment:
Our schema has no concept of a Book, but we probably actually want to implement that here in the app data model. Projects should be limited to containing a Book (which can extend collection, and be stored as a collection in the database). Projects shouldn't really operate on the Anthology (or higher) or Chapter (or lower) level after all; given how our UI hierarchy works.